apps/ledger-live-desktop/src/renderer/screens/exchange/Swap2/Form/index.tsx

```diff
@@ -118,15 +118,24 @@ const SwapForm = () => {
   }, []);
 
   const exchangeRatesState = swapTransaction.swap?.rates;
-  const swapError = swapTransaction.fromAmountError || exchangeRatesState?.error;
+  let swapError = swapTransaction.fromAmountError || exchangeRatesState?.error;
```

Comment:
for readability of the code : 
- having "const" is better, I know it's definitive.
- I like to read small functions

I suggest you let const here, ending with || detectSpecificCoinError()
detectSpecificCoinError() may switch on the family first

Reply:
Yeah you're right, extracted tezos check in its own function